### DISCUSSION TITLE
How to download the subtitles in english?

### DISCUSSION THREAD

**Discussion by bheeshmpita** [JunkFood02/Seal#908] [Q&A]
While downloading the video, i enable the subtitles tab and uner the subtitles langauage settings i tried 
```
--sub-langs en
--sub-langs "en"
--sub-langs "en.*"
```
but none of them works and while downloading video in the notification bar under the list of the commands that gets processed there appears "There are no subtitles for the requested language" which is certainly not the case as i did verify that subtitles were available and i tried it on several videos but none of them worked and even auto generated ones dont work inspite of enabling its tab.
What am i missing?

**Comment by JunkFood02** [comment]
It seems like you're using custom commands, you should first use the built-in options for downloading the video

**Reply by bheeshmpita** [reply]
Thanks, can you please explain a little bit in detail what should i exactly for getting subtitles downloaded along with the video. I don't face any errors while downloading videos.

**Reply by Marco-9456** [reply]
<details>
<summary>Screen record</summary>

https://github.com/JunkFood02/Seal/assets/41828058/511791a6-2bfe-4f28-9044-85fe264e693e

</details>

**Reply by Pierack** [reply]
> Screen record

Hi @Marco-9456 I would like to know how I can select two languages.
Would this be ok? The .es is for Spanish because I want to download subtitles in that language.

en.*,es.*,.*-orig

**Reply by Marco-9456** [reply]
> > Screen record
> 
> Hi @Marco-9456 I would like to know how I can select two languages. Would this be ok? The .es is for Spanish because I want to download subtitles in that language.
> 
> en._,es._,.*-orig

Downloading subtitles is much easier now in the newer versions of the app:

https://github.com/user-attachments/assets/0c45d488-35a5-408c-8e5e-5112bf1ff9de
